pkgs/by-name/wa/waybar/package.nix

```diff
@@ -153,6 +155,7 @@ stdenv.mkDerivation (finalAttrs: {
       "cava" = cavaSupport;
       "dbusmenu-gtk" = traySupport;
       "jack" = jackSupport;
+      "libevdev" = evdevSupport;
```

Comment:
this is already in the derivation... conflict resolution issue / rebase needed? https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/wa/waybar/package.nix#L156